utils/metrics/create_confusion_matrix.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+
+def create_confusion_matrix(prediction: list, true_y: list, save_location: str):
+    """
+    This function makes confusion matrix and saves it 
+    """
+    # Create confusion matrix in pandas
+    results = pd.DataFrame({"prediction": prediction, "expected": true_y})
+    results["combined"] = results["prediction"] + "_" + results["expected"]
```

Comment:
```suggestion
    results = results.assign(combined=results.loc[:, "prediction"] + "_" results.loc[:, "expected"])
```

For creating new variables, I tend to prefer [`.assign()`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.assign.html). Assign works well with [method chaining](https://tomaugspurger.github.io/method-chaining.html) and its good to get in that habit